core/extraction/towns_fund.py

```diff
@@ -324,12 +323,14 @@ def extract_project(df_project: pd.DataFrame, project_lookup: dict, programme_id
     df_project["Postcodes"] = df_project.apply(
         lambda row: ",".join(extract_postcodes(row["Locations"])), axis=1
     )  # TODO: keep as a list and store as array of strings in db (not supported by SQLite)
+
     single_lat_long = "Single location __Project Location - Lat/Long Coordinates (3.d.p e.g. 51.496, -0.129)"
     multiple_lat_long = "Multiple locations __Project Locations - Lat/Long Coordinates (3.d.p e.g. 51.496, -0.129)"
     df_project["Lat/Long"] = df_project.apply(
         lambda row: row[single_lat_long] if row["Single or Multiple Locations"] == "Single" else row[multiple_lat_long],
         axis=1,
     )
+    df_project["Postcodes"] = df_project["Postcodes"].replace("", np.nan, regex=True)
```

Comment:
Why have you set regex=True? If you're just replacing empty strings I don't think it's necessary.

Reply:
oops, left that in by mistake, good spot